src/unlock/signer.rs

```diff
@@ -503,6 +596,16 @@ impl ScriptSigner for OmniLockScriptSigner {
         }
         if self.config.id.flag == IdentityFlag::PubkeyHash {
             self.signer.match_id(self.config.id.blake160.as_ref())
+        } else if self.config.id.flag == IdentityFlag::Multisig {
```

Comment:
Use `match self.config.id.flag {}` would be better, since `if else` may missing some `enum IdentityFlag` variant.